gui/default/syncthing/core/syncthingController.js

```diff
@@ -584,6 +624,21 @@ angular.module('syncthing.core')
             }).error($scope.emitHTTPError);
         }
 
+        $scope.probeRemoteGUIAddress = function (deviceId, address) {
+            $http({
+                method: "OPTIONS",
+                url: address,
+            }).success(function (data) {
+                $scope.remoteGUICache[address] = true;
+                $scope.idToRemoteGUI[deviceId] = address;
+                console.log("probeRemoteGUIAddress", data);
+            }).error(function (err) {
+                $scope.remoteGUICache[address] = false;
+                $scope.idToRemoteGUI[deviceId] = "";
```

Comment:
I don't know if this extensive logging makes sense, I had just added it for debugging the code.